test/test_suit/test_nullable_values_after_smt.py

```diff
@@ -0,0 +1,59 @@
+import json
+from snowflake.connector import DictCursor
+from test_suit.test_utils import NonRetryableError, RetryableError
+
+
+class TestNullableValuesAfterSmt:
+
+    def __init__(self, driver, nameSalt):
+        self.driver = driver
+        self.fileName = 'travis_correct_nullable_values_after_smt'
+        self.table = self.fileName + nameSalt
+        self.topic = self.table + '0'
+
+        self.driver.snowflake_conn.cursor().execute(
+            f'create or replace table {self.table} (index number not null)')
+
+        self.driver.snowflake_conn.cursor().execute(
+            f'alter table {self.table} set enable_schema_evolution = true')
+
+    def getConfigFileName(self):
+        return self.fileName + '.json'
+
+    def send(self):
+        value = []
+        for idx in range(200):
```

Comment:
Is there a specific reason why we send 200? Because of buffering? Why not more or less? It would be best to send the minimal required amount to keep the processing short.

Reply:
No, there is no specific reason. However, the number of *ingested* events must be lower or equal to `buffer.count.records` parameter. 
We can decrease a number of events to 100 or even 20, but most of the test time is spent for `=== Sleep 60 secs before verify result in Snowflake DB ===` anyway.